### ISSUE TITLE
Use a logging library

### ISSUE THREAD

**Issue by chamilad** [abhishek0198/wso2dockerfiles-test-framework#23]
It might be better to use a third party logging library to manage log levels, rather than manually prefixing log levels to output. It might also enable handling of log outputs to different channels (ex: screen, log file) differently. 

http://dave.cheney.net/2015/11/05/lets-talk-about-logging

**Comment by imesh** [comment]
Yes, -1 for manually handling log levels without using a logging framework.
